tests/__test__/scheduler.test.js

```diff
@@ -0,0 +1,161 @@
+describe('Basic user flow for Website', () => {
+    // First, visit the scheduler website
+    beforeAll(async () => {
+      await page.goto('https://cse110-fa22-group10.github.io/cse110-fa22-group10/scheduler/index.html');
+      jest.setTimeout(10000)
+    });
+  
+    // Next, check to make sure that all 0 <post-card> elements have loaded
+    it('Initial Home Page - Check for 0 post cards', async () => {
+      console.log('Checking for 0 post cards...');
+      // Query select all of the <post-card> elements and return the length of that array
+      const numCards = await page.$$eval('post-card', (postCards) => {
+        return postCards.length;
+      });
+      // Expect there that array from earlier to be of length 0, meaning 0 <post-card> elements where found
+      expect(numCards).toBe(0);
+    }, 10000);
```

Comment:
is it reasonable for the time limit to be 10 seconds or should we expect it to be faster?